datastore/FertiScan/__init__.py

```diff
@@ -1,6 +1,12 @@
 import os
```

Comment:
I suggest using lowercase for folder names. Check [this](https://stackoverflow.com/questions/52827722/folder-naming-convention-for-python-projects).